src/app/tabs/test-tab.js

```diff
@@ -141,37 +145,64 @@ module.exports = class TestTab extends ViewPlugin {
-      _errors.errors.forEach((err) => this.renderer.error(err.formattedMessage || err.message, this.testsSummary, {type: err.severity}))
+      _errors.errors.forEach((err) => this.renderer.error(err.formattedMessage || err.message, this.testsOutput, {type: err.severity}))
     } else if (_errors && Array.isArray(_errors) && (_errors[0].message || _errors[0].formattedMessage)) {
-      _errors.forEach((err) => this.renderer.error(err.formattedMessage || err.message, this.testsSummary, {type: err.severity}))
+      _errors.forEach((err) => this.renderer.error(err.formattedMessage || err.message, this.testsOutput, {type: err.severity}))
     } else if (_errors && !_errors.errors && !Array.isArray(_errors)) {
       // To track error like this: https://github.com/ethereum/remix/pull/1438
-      this.renderer.error(_errors.formattedMessage || _errors.message, this.testsSummary, {type: 'error'})
+      this.renderer.error(_errors.formattedMessage || _errors.message, this.testsOutput, {type: 'error'})
     }
-    this.testsSummary.appendChild(yo`<div class=${css.summaryTitle}> ${filename} </div>`)
     yo.update(this.resultStatistics, this.createResultLabel())
     if (result) {
-      if (result.totalPassing > 0) {
-        this.testsSummary.appendChild(yo`<div class="text-success">${result.totalPassing} passing (${result.totalTime}s)</div>`)
-        this.testsSummary.appendChild(yo`<br>`)
-      }
-      if (result.totalFailing > 0) {
-        this.testsSummary.appendChild(yo`<div class="text-danger">${result.totalFailing} failing</div>`)
-        this.testsSummary.appendChild(yo`<br>`)
+      if (result.totalPassing > 0 && result.totalFailing > 0) {
+        this.testsOutput.appendChild(yo`<div class="text-success">${result.totalPassing} passing, <span class="text-danger"> ${result.totalFailing} failing </span> (${result.totalTime}s)</div>`)
+        if (this.rawFileName === filename) this.setHeader(false)
+      } else if (result.totalPassing > 0 && result.totalFailing <= 0) {
+        this.testsOutput.appendChild(yo`<div class="text-success">${result.totalPassing} passing (${result.totalTime}s)</div>`)
+        if (this.rawFileName === filename) this.setHeader(true)
+      } else if (result.totalPassing <= 0 && result.totalFailing > 0) {
+        this.testsOutput.appendChild(yo`<div class="text-danger">${result.totalFailing} failing</div>`)
+        if (this.rawFileName === filename) this.setHeader(false)
       }
+      const displayError = yo`<div class="sol error alert alert-danger"></div>`
+
       result.errors.forEach((error, index) => {
-        this.testsSummary.appendChild(yo`<div class="text-danger">${error.context} - ${error.value} </div>`)
-        this.testsSummary.appendChild(yo`<div class="${css.testFailureSummary} text-danger" >${error.message}</div>`)
-        this.testsSummary.appendChild(yo`<br>`)
+        displayError.appendChild(yo`<div>
+        <ul type="disc" class="ml-3 mb-0"><li>${error.value} </li></ul>
```

Comment:
why is this type?

Reply:
It sets the shape of the bullets for the `ul` list. I will take it out also because by default the type is "disc"